formatter.go

```diff
@@ -30,7 +38,7 @@ type Formatter interface {
 //
 // It's not exported because it's still using Data in an opinionated way. It's to
 // avoid code duplication between the two default formatters.
-func prefixFieldClashes(data Fields) {
+func prefixFieldClashes(data Fields, hasRemoteCaller bool) {
```

Comment:
It seems safe enough to always prefix the caller key, which would avoid the need to change the method signature and the calls in the formatters.